caravel/assets/javascripts/SqlLab/components/ResultSet.jsx

```diff
@@ -72,7 +72,7 @@ class ResultSet extends React.Component {
           <div className="ResultSet">
             <Table
               data={results.data}
-              columns={results.columns}
+              columns={results.columns.map(function (o) { return o.name; })}
```

Comment:
for very short function you should use the more concise arrow operator:
`(col) => coll.name`